content/docs/contributing/policy.md

```diff
@@ -3,59 +3,109 @@ title: Feature Policy
 description: 'cert-manager contributing guide: Contribution Policy'
 ---
 
-We are open to feature requests and PRs implementing those. If you plan on contributing a new feature into cert-manager we recommend creating an issue first for it to be discussed with the cert-manager maintainers. Another possibility is bringing it up in a community meeting discussion for an open discussion with the maintainers and community on the implementation.
+We love to receive both feature requests and PRs which add stuff to cert-manager; the community is at the heart of what we do!
 
-# Features we will not allow
+If you're thinking of adding a feature, we recommend you read this doc to maximize the chances of your contribution getting the attention it deserves and hopefully to get it merged quickly!
 
-Certain features have been requested before, but have not been implemented properly or not able to be implemented without breaking the security model.
+We recommend creating an issue first for it to be discussed with the cert-manager maintainers. Another possibility is bringing it up in a community meeting discussion for an open discussion with the maintainers and community on the implementation.
 
-Below is a list of a few of these:
+## Feature Sizing: Getting Your Change Accepted
 
-## Vendoring Kubernetes related APIs outside of the `k8s.io/` namespace
+We evaluate new features and PRs based on their size; either they're small or large.
+
+### Smaller Features
+
+Many contributions are small. That usually - but not always - means that implementing them won't require many lines of code to be added or changed, and in any case they should be easy
+for maintainers to review. A PR being small is a good thing; if you can down-scope your feature to make it smaller, we won't complain!
+
+If you believe your feature is small, please feel free to just raise a PR and optionally also post a link to your PR in the [cert-manager-dev slack channel](./README.md#slack). Usually a sufficiently small PR can be merged without too much ceremony. If we think it's actually a larger piece of work, we'll let you know.
+
+### Larger Features
+
+If you're not sure whether your PR is small, or if you know it's bigger, you'll want to speak to us first before raising a PR. This
+will help to ensure that your PR is something we're likely to merge to avoid wasting your time. It'll also make it easier
+for us to do the design process.
+
+Larger PRs should normally come with a design document. This doesn't need to be a huge piece of work; rather, it should specify
+_why_ we'd want a given feature, _how_ it'll be implemented, and perhaps evaluate pros and cons. You can add this design in a
+GitHub [feature request](https://github.com/cert-manager/cert-manager/issues/new?assignees=&labels=&template=feature-request.md) or [bug report](https://github.com/cert-manager/cert-manager/issues/new?assignees=&labels=&template=bug.md).
```

Comment:
Question: does this part essentially suggest that a design should be PRed to https://github.com/cert-manager/cert-manager/tree/master/design? (Which I thought was what we wanted)

Reply:
I guess this is also where the design doc template would come in as there would be some generic things that all docs such docs should include (i.e whether this change involves an API change, whether it will be feature gated etc)

Reply:
It doesn't suggest that - it just says "raise an issue". I think PRing to the design folder might be a better shout, though.

I basically just suggested the thing that was the least friction for people here. But having this stuff versioned and reviewed through a PR is probably preferable! I'll update.